tsdb/db.go

```diff
@@ -52,6 +52,9 @@ const (
 	// Default duration of a block in milliseconds.
 	DefaultBlockDuration = int64(2 * time.Hour / time.Millisecond)
 
+	// Interval for scheduled compaction for OOO head
+	OOOCompactInterval = 2 * time.Hour
```

Comment:
Can you make this configurable?  Perhaps it can use the same interval as regular compaction?

Reply:
Plus this configuration can help in testing